index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Alzubair Protfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="container">
+    <h1 class="navbar">My Logo</h1>
+    <i class="fa-solid fa-bars"></i>
+  </header>
+
+  <div class="body">
+    <img src="Microverse-Student-Project-1/img.png" alt="background-img">
+    <div class="paragraph">
+      <h1 class="first-text">I’m Alzubair Glad to see you!</h1>
+      <h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
+      <a href="#">LET’S CONNECT</a>
+      <br>
+      <i class="emoji fa-brands fa-twitter"></i>
+      <i class="emoji fa-brands fa-linkedin"></i>
+      <i class="emoji fa-solid fa-m"></i>
+      <i class="emoji fa-brands fa-github"></i>
+      <i class="emoji fa-solid fa-hand-peace"></i>
```

Comment:
- [ ] please ensure that your social icons are inside a `ul tag` and each of them in a `li tag`.
- [ ] Add link place holders to each icon ..`<a href="/"></a>` .you can check how to add that [here](https://www.w3schools.com/html/html_links.asp)